tests/e2e/support.py

```diff
@@ -163,3 +163,31 @@ def create_kueue_resources(
             body=local_queue_json,
         )
         print(f"'{local_queue}' created in namespace '{self.namespace}'")
+
+
+def delete_kueue_resources(
+    self, cluster_queue="cluster-queue-mnist", flavor="default-flavor-mnist"
+):
+    # Delete if given resource-flavor exists
+    try:
+        self.custom_api.delete_cluster_custom_object(
+            group="kueue.x-k8s.io",
+            plural="resourceflavors",
+            version="v1beta1",
+            name=flavor,
+        )
+        print(f"'{flavor}' resource-flavor deleted")
+    except Exception as e:
+        print(f"Error deleting resource-flavor '{flavor}' : {e}")
+
+    # Delete if given cluster-queue exists
+    try:
+        self.custom_api.delete_cluster_custom_object(
```

Comment:
Help me remember: on behalf of which user these operations are performed?

Reply:
I suppose it is https://github.com/project-codeflare/codeflare-sdk/blob/main/.github/workflows/e2e_tests.yaml#L109-L125

Reply:
Right. Ideally there would be two users impersonating the two different personas, SDK user on one hand, RHOAI admin on the other hand, and the latter would perform operations related to queue management.